templates/pimcore5/files/.platform/services.yaml

```diff
@@ -0,0 +1,18 @@
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Platform.sh project.
+pimcore_mysqldb:
+  type: mysql:10.2
+  disk: 2048
+  configuration:
+    schemas:
+      - project_database
+    endpoints:
+      pimcore_admin:
+        default_schema: project_database
+        privileges:
+          project_database: admin
```

Comment:
None of this custom configuration should be necessary, unless Pimcore only works with that database name.  (In which case it would be very dumb.)  It can probably just be entirely omitted.  (If there's a specific reason it has to be explicit, that needs to be documented in comments.)

Reply:
Indeed they are not needed.